examples/with-stripe-typescript/app/donate-with-checkout/result/page.tsx

```diff
@@ -0,0 +1,32 @@
+import Stripe from 'stripe'
+
+import PrintObject from '../../components/PrintObject'
+
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!, {
```

Comment:
I wonder if all of the stripe logic should be pulled into `lib/stripe` and then add `import "server-only"` given this a secret key, for best practice. 

https://nextjs.org/docs/getting-started/react-essentials#keeping-server-only-code-out-of-client-components-poisoning